corpysprofiling/corpysprofiling.py

```diff
@@ -114,12 +117,46 @@ def corpus_viz(corpus, stats):
     Examples
     --------
     >>> from corpysprofiling import corpysprofiling
-    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 'words')
-    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 'topic')
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")['word cloud']
+    >>> plt.figure()
+    >>> plt.imshow(wordcloud, interpolation="bilinear")
+    >>> plt.axis("off")
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")['df used for bar']
+    >>> df.plot.bar(rot=0, x='words')
+    >>> plt.xticks(rotation=90)
+    >>> plt.xlabel("Words")
     >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 15)
     TypeError: Input must be a string
     """
-    return
+    # Step 1. To get a word cloud
+    wordcloud = WordCloud().generate(corpus.lower())
+    plt.figure()
+    plt.imshow(wordcloud, interpolation="bilinear")
```

Comment:
This is printing the wordcloud directly to the screen.   
Suggest adding `plt.close(1)` after `plt.axis("off")`